saltyrtc/server/server.py

```diff
@@ -246,7 +246,7 @@ def handler(self, connection, ws_path):
                         AddressType.server, responder_id, client.id)
                     responder.log.debug('Enqueueing disconnected message')
                     coroutines.append(responder.enqueue_task(responder.send(message)))
-                yield from asyncio.gather(*coroutines)
+                yield from asyncio.gather(*coroutines, loop=self._loop)
```

Comment:
@lgrahl like this? Sorry, I just assumed that it would use the default loop.

Reply:
The quirks of asyncio (it should probably be a method on the `loop`).